diff --git a/node_modules/@kitware/vtk.js/Common/Core/DataArray.d.ts b/node_modules/@kitware/vtk.js/Common/Core/DataArray.d.ts
index c9fbaa5..8b31d56 100644
--- a/node_modules/@kitware/vtk.js/Common/Core/DataArray.d.ts
+++ b/node_modules/@kitware/vtk.js/Common/Core/DataArray.d.ts
@@ -54,7 +54,7 @@ export interface vtkDataArray extends vtkObject {
 	 *
 	 * @param {Number} componentIndex (default: -1)
 	 */
-	getRange(componentIndex?: number): vtkRange;
+	getRange(componentIndex?: number): [number, number];
 
 	/**
 	 *
diff --git a/node_modules/@kitware/vtk.js/Rendering/Core/RenderWindow.d.ts b/node_modules/@kitware/vtk.js/Rendering/Core/RenderWindow.d.ts
index fda2089..382c406 100644
--- a/node_modules/@kitware/vtk.js/Rendering/Core/RenderWindow.d.ts
+++ b/node_modules/@kitware/vtk.js/Rendering/Core/RenderWindow.d.ts
@@ -53,7 +53,7 @@ export interface vtkRenderWindow extends vtkObject {
 	 * @param {String} format 
 	 * @param {*} opts 
 	 */
-	captureImages(format?: string, opts?: any): void;
+	captureImages(format?: string, opts?: any): Promise<string>[];
 
 	/**
 	 * 
@@ -123,7 +123,7 @@ export interface vtkRenderWindow extends vtkObject {
 	 * @param name 
 	 * @param initialValues 
 	 */
-	newAPISpecificView(name: string, initialValues: object): any;
+	newAPISpecificView(name?: string, initialValues?: object): any;
 
 	/**
 	 * Remove renderer
diff --git a/node_modules/@kitware/vtk.js/Rendering/Core/VolumeProperty.d.ts b/node_modules/@kitware/vtk.js/Rendering/Core/VolumeProperty.d.ts
index f8304b7..eec495d 100644
--- a/node_modules/@kitware/vtk.js/Rendering/Core/VolumeProperty.d.ts
+++ b/node_modules/@kitware/vtk.js/Rendering/Core/VolumeProperty.d.ts
@@ -263,9 +263,9 @@ export interface vtkVolumeProperty extends vtkObject {
 	/**
 	 *
 	 * @param {Number} index 
-	 * @param {Number} value
+	 * @param {Boolean} value
 	 */
-	setUseGradientOpacity(index: number, value: number): boolean;
+	setUseGradientOpacity(index: number, value: boolean): boolean;
 
 	/**
 	 *
diff --git a/node_modules/@kitware/vtk.js/Widgets/Core/AbstractWidgetFactory.js b/node_modules/@kitware/vtk.js/Widgets/Core/AbstractWidgetFactory.js
index f990caa..b1706d0 100644
--- a/node_modules/@kitware/vtk.js/Widgets/Core/AbstractWidgetFactory.js
+++ b/node_modules/@kitware/vtk.js/Widgets/Core/AbstractWidgetFactory.js
@@ -189,17 +189,16 @@ function vtkAbstractWidgetFactory(publicAPI, model) {
     return unsubscribe();
   }); // Defer after object instantiation so model.widgetState actually exist
 
-  setTimeout(function () {
     if (model.widgetState) {
       unsubscribe = model.widgetState.onModified(function () {
         return publicAPI.invokeWidgetChange(model.widgetState);
       }).unsubscribe;
     }
-  }, 0);
 } // ----------------------------------------------------------------------------
 
 
-function extend(publicAPI, model) {
+function extend(publicAPI, model, initialValues = {}) {
+Object.assign(model, initialValues);
   macro.obj(publicAPI, model);
   macro.get(publicAPI, model, ['widgetState']);
   macro.event(publicAPI, model, 'WidgetChange');
